packages/atomic/src/components/common/checkbox.spec.ts

```diff
@@ -0,0 +1,125 @@
+import {fireEvent, within} from '@storybook/test';
+import {html, render} from 'lit';
+import {vi} from 'vitest';
+import {checkbox, CheckboxProps} from './checkbox';
+
+describe('checkbox', () => {
+  let container: HTMLElement;
+
+  beforeEach(() => {
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  afterEach(() => {
+    document.body.removeChild(container);
+  });
+
+  const renderCheckbox = (props: Partial<CheckboxProps>): HTMLButtonElement => {
+    render(
+      html`${checkbox({
+        ...props,
+        checked: props.checked ?? false,
+        onToggle: props.onToggle ?? vi.fn(),
+      })}`,
+      container
+    );
+    return within(container).getByRole('checkbox') as HTMLButtonElement;
+  };
+
+  it('should render a checkbox with the correct text attributes', () => {
```

Comment:
I think those should be tested separately, and we should also test that `aria-label` will fallback to the value of the `text` prop if the `ariaLabel` prop isn't specified